bottlecap/src/proxy/interceptor.rs

```diff
@@ -153,6 +160,18 @@ async fn invocation_next_proxy(
             }
         };
 
+    if let Some(appsec) = appsec_processor {
+        if let Some(rid) = intercepted_parts
```

Comment:
```suggestion
        if let Some(request_id) = intercepted_parts
```

Just to keep consistency in this file